source/binding/OperatorExpressions.cpp

```diff
@@ -1510,14 +1511,16 @@ Expression& StreamingConcatenationExpression::fromSyntax(
 
     auto badResult = [&]() {
         return compilation.emplace<StreamingConcatenationExpression>(
-            compilation.getErrorType(), sliceSize, span<const Expression*>(), syntax.sourceRange());
+            compilation.getErrorType(), sliceSize, span<const StreamExpression*>(),
+            syntax.sourceRange());
     };
 
     if (!(context.flags & BindFlags::StreamingAllowed)) {
         context.addDiag(diag::BadStreamContext, syntax.operatorToken.location());
         return badExpr(compilation, badResult());
     }
 
+    // slice_size
```

Comment:
Can you expand on this comment? It doesn't seem to be adding much in its current form.

Reply:
I insert some important quotes from IEEE standard at places around relevant code.
```c++
       // The slice_size determines the size of each block. If specified, it may be a constant
       // integral expression or a simple type.

                // It shall be an error for the value to be zero or negative.

          // Left-to-right streaming using >> shall cause the slice_size to be ignored and no
          // re-ordering performed.
```